go/chat/teams.go

```diff
@@ -634,14 +631,9 @@ func (t *ImplicitTeamsNameInfoSource) LookupName(ctx context.Context, tlfID chat
 		return res, err
 	}
 	t.Debug(ctx, "LookupName: got name: %s", impTeamName.String())
-	idFailures, err := t.identify(ctx, tlfID, impTeamName)
-	if err != nil {
-		return res, err
-	}
 	return types.NameInfo{
-		ID:               tlfID,
-		CanonicalName:    impTeamName.String(),
-		IdentifyFailures: idFailures,
```

Comment:
can you remove this field from the struct too?

Reply:
Yeah I guess nothing reads from it, I'll drop it.